packages/pipeline/src/data_sources/ohlcv_external/crypto_compare.ts

```diff
@@ -33,23 +33,28 @@ export interface CryptoCompareOHLCVParams {
     toTs?: number;
 }
 
-const ONE_WEEK = 7 * 24 * 60 * 60 * 1000; // tslint:disable-line:custom-no-magic-numbers
 const ONE_HOUR = 60 * 60 * 1000; // tslint:disable-line:custom-no-magic-numbers
 const ONE_SECOND = 1000;
 const ONE_HOUR_AGO = new Date().getTime() - ONE_HOUR;
 const HTTP_OK_STATUS = 200;
 const CRYPTO_COMPARE_VALID_EMPTY_RESPONSE_TYPE = 96;
+const MAX_PAGE_SIZE = 2000;
 
 export class CryptoCompareOHLCVSource {
-    public readonly interval = ONE_WEEK; // the hourly API returns data for one week at a time
-    public readonly default_exchange = 'CCCAGG';
     public readonly intervalBetweenRecords = ONE_HOUR;
+    public readonly default_exchange = 'CCCAGG';
```

Comment:
Shoot I'm sorry I missed this before, but this variable should be in `camelCase`.

Reply:
This is the cryptocompare name for the exchange, this is their internal "aggregate" across exchanges. maybe lower case? but otherwise i think cccagg or CCCAGG

Reply:
ohh I totally missed this! thanks for catching.